Source/controls/keymapper.cpp

```diff
@@ -19,7 +19,11 @@ Keymapper::Keymapper(SetConfigKeyFunction setKeyFunction, GetConfigKeyFunction g
     , getKeyFunction(getKeyFunction)
 {
 	// Insert all supported keys: a-z, 0-9 and F1-F12.
+#ifndef __vita__
+	// HACK: Disable reserving on Vita because it triggers a linking error:

```

Comment:
"Fix Vita build" :joy_cat:

Reply:
Well, we already found proper fix (remove libm from link). @glebm said he'll pullrequest it

Reply:
I said I'd pull request the SDL_ttf but I'm not sure how to fix the libm thing

Reply:
Aaah. Well, dropping libm from linked libs seems to help (it's still linked by sdl target)